scripts/tdp-release-uris.txt

```diff
@@ -36,4 +36,5 @@ https://github.com/TOSIT-FR/tez/releases/download/tez-0.9.1-TDP-0.1.0-SNAPSHOT/t
 https://dlcdn.apache.org/lucene/solr/7.7.3/solr-7.7.3.tgz
 # zookeeper
 https://archive.apache.org/dist/zookeeper/zookeeper-3.4.6/zookeeper-3.4.6.tar.gz
-
+# livy
```

Comment:
We're removing these lines as part of https://github.com/TOSIT-IO/tdp-getting-started/pull/74.

We should also do it here